### DISCUSSION TITLE
Black Amoled Background

### DISCUSSION THREAD

**Discussion by ghost** [WrichikBasu/ShakeAlarmClock#87] [Feature requests]
Hey,

i selected as theme dark in the settings but it is still not really dark...Could you please add Black Amoled Background? Meaning pure Black that the App really gets the most dark in total. Because I tried system theme too but even tho I have pure Black in my Systeme, it does not change...I hope this will be fixed.

**Comment by WrichikBasu** [comment]
I'll have a look.

**Reply by ghost** [reply]
Thank you. I added two Screenshots. Dark for me would be that the blue at the top would be gray too.

![IMG_20241121_175406](https://github.com/user-attachments/assets/86b949e9-0d27-47a3-ab67-14d0fbcb4f87)
![IMG_20241121_175353](https://github.com/user-attachments/assets/8a03a3f7-114f-42c2-94ae-8cc2ac6422b0)
